hawc/apps/riskofbias/templates/riskofbias/rob_edit.html

```diff
@@ -1,9 +1,7 @@
 {% extends 'assessment-rooted.html' %}
 
-{% load add_class %}
 {% load get_at_index %}
```

Comment:
I dont think this is used here, or anywhere else? delete `get_at_index` from this template and the `study/templatetags`?